src/snark/src/zk_proof_systems/ppzksnark/r1cs_ppzksnark/r1cs_ppzksnark.tcc

```diff
@@ -451,81 +521,36 @@ r1cs_ppzksnark_proof<ppT> r1cs_ppzksnark_prover(const r1cs_ppzksnark_proving_key
     const qap_witness<Fr<ppT> > qap_wit = r1cs_to_qap_witness_map(constraint_system, primary_input, auxiliary_input, d1, d2, d3);
     leave_block("Compute the polynomial H");
 
-#ifdef DEBUG
-    const Fr<ppT> t = Fr<ppT>::random_element();
-    qap_instance_evaluation<Fr<ppT> > qap_inst = r1cs_to_qap_instance_map_with_evaluation(constraint_system, t);
-    assert(qap_inst.is_satisfied(qap_wit));
-#endif
```

Comment:
Is dropping this debug assertion intentional?

Reply:
@ebfull ?

Reply:
It was intentional when we did it. I can't remember the reason why though. It could be left in -- after all, we don't enable `DEBUG` anyway.

Reply:
added back